fastestimator/op/op.py

```diff
@@ -128,4 +128,4 @@ def __init__(self,
         self.in_list = True
 
     def forward(self, data: List[Any], state: Dict[str, Any]) -> Any:
-        return self.fn(*data)
+        return self.fn(*to_list(data))
```

Comment:
no need to call to_list when in_list = True

Reply:
When inputs is None, the data becomes None.

Reply:
that's a problem - should be resolved elsewhere such that in_list = True always results in a list being passed

Reply:
like this?